frontends/web/src/utils/component.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentConstructor, FunctionalComponent } from 'preact';
+
+export function getDisplayName(component: ComponentConstructor<any> | FunctionalComponent<any>): string {
+    return component.displayName || (component as any).name || 'Component';
+}
```

Comment:
what is the purpose of utils/component.ts ?

Reply:
to answer my own question (again :) ) https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging

Reply:
yes I saw that after looking at the code :)